ajax/libs/jquery-spellchecker/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "jquery-spellchecker",
+  "filename": "jquery.spellchecker.min.js",
+  "description": "A lightweight jQuery plugin that can check the spelling of text within a form field or DOM tree.",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/badsyntax/jquery-spellchecker"
+  },
+  "keywords": [
+    "jquery",
+    "spellcheck"
+  ],
+  "version": "0.2.4-ignored",
+  "npmName": "jquery-spellchecker",
+  "npmFileMap": [
+    {
```

Comment:
The library only from GitHub, not npm.
So we use git auto-update. Let's modify from A to B.

- A
```js
  "npmName": "jquery-spellchecker",
  "npmFileMap": [
    {
      "basePath": "dist/js",
      "files": [
        "**/*"
      ]
    }
  ]
```

- B
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/badsyntax/jquery-spellchecker.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "+(js|css)/jquery.spellchecker*"
        ]
      }
    ]
  }
```

Thank you.